meal-planner/src/hooks/useRecipe.js

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from 'react'
+import mealPlan from '../data/MealPlan'
+import loadLocally from '../lib/loadLocally'
+import saveLocally from '../lib/saveLocally'
```

Comment:
You could make a localStorage.js and put both functions in it, just to simplify things

Reply:
jesus, your'e critical!